.env.development

```diff
@@ -36,3 +36,4 @@ STRIPE_RESPONSE_URL=http://localhost:18130/payment/stripe/checkout/
 STRIPE_DEFERRED_INTENT_BETA_FLAG=elements_enable_deferred_intent_beta_1
 SUBSCRIPTIONS_BASE_URL='http://localhost:18750'
 ENABLE_B2C_SUBSCRIPTIONS=true
+LEARNER_SUPPORT_URL=https://support.edx.org/hc/en-us/articles/12975352138007
```

Comment:
Can we remove the /en-us/ from the URL, this would ensure the support.edx.org picks up the right language automatically for the article

Reply:
done.